Plugins~/Src/MeshSyncClientBlender/python/2.90.1/unity_mesh_sync.py

```diff
@@ -15,11 +15,6 @@
 from .unity_mesh_sync_common import *
 from .unity_mesh_sync_preferences import *
 
-class MESHSYNC_PT:
```

Comment:
This was duplicated across all blender version files and I need it in another module, so I moved this to the unity_mesh_sync_common class.